content/Proposals/Done/202106-proposals-process.md

```diff
@@ -0,0 +1,112 @@
+# 2021-06: Proposal Process
```

Comment:
Shouldn't this document be in the `Proposals/In Review` folder?

Reply:
No, we do that only for Google Docs. For markdowns we know it's in progress since it's in PR (: WDYT?

Reply:
Ah got it, makes complete sense :+1: